client/scss/utilities/_mixins.scss

```diff
@@ -36,14 +36,13 @@
   $to-list: null;
   $between-list: null;
 
-  @each $sizes in $cq-list {
-    $size-key: nth($sizes, 1);
+  @each $size-key, $sizes-value in $cq-list {
```

Comment:
Maybe this would be better as `$size-value` since we're using `$size-key`, for consistency.